Servers/SystemServer/main.cpp

```diff
@@ -8,26 +8,26 @@
 #include <sys/types.h>
 #include <unistd.h>
 
-void start_process(const char* prog, const char* arg, int prio, const char* tty = nullptr)
+void start_process(String program, Vector<String> arguments, int prio, const char* tty = nullptr)
```

Comment:
No need to make copies of these:
```suggestion
void start_process(const String& program, const Vector<String>& arguments, int prio, const char* tty = nullptr)
```